resources/dicts/relationship_events/normal_interactions/neutral.json

```diff
@@ -13,29 +13,28 @@
 			"m_c and r_c notice a cloud is shaped like a bird.",
 			"m_c points out a constellation to r_c.",
 			"m_c and r_c keep trying to figure out what a bad smell in camp is.",
-			"m_c poses a crazy theory to r_c and they chat about it for a bit.",
-			"The whole Clan sees m_c and r_c accidentally headbutt each other. Embarrassing.",
+			"m_c poses a crazy theory to r_c, and they chat about it for a while.",
 			"m_c asks r_c if there's something in {PRONOUN/m_c/poss} teeth.",
 			"m_c points out a nice rock for r_c to sharpen {PRONOUN/r_c/poss} claws on.",
-			"m_c and r_c take a moment to fill a puddle in the camp with sand together.",
+			"m_c and r_c fill a puddle in the camp with sand together.",
 			"m_c notices some of r_c's fur on {PRONOUN/m_c/poss} fresh-kill.",
 			"Both m_c and r_c startle at a clap of thunder.",
 			"m_c pretends not to see r_c cough up a hairball.",
 			"m_c and r_c are assigned the same task.",
 			"m_c and r_c accidentally startle one another.",
 			"m_c runs into r_c at the medicine den. They both have a tummy ache from two halves of the same prey.",
-			"m_c and r_c get their tails stuck together with some sap.",
+			"m_c and r_c get their fur stuck together with some sap.",
 			"r_c smacks a spider off of m_c.",
 			"r_c has to help m_c stop choking on a huge bone.",
 			"m_c and r_c chat idly about the latest meeting.",
 			"m_c and r_c keep making each other yawn.",
-			"m_c and r_c realize to their horror that there's a dead animal in the puddle they are drinking from.",

```

Comment:
Removed

Reply:
Thought this one was a bit too unpleasant. ^^;